### PR TITLE
Save ini params to json file

### PR DESCRIPTION
**Pull request by VHolonec**

### CODE REVIEW

File: sdk/include/aditof/camera.h [hunk 1]
```diff
@@ -534,6 +534,16 @@ class SDK_API Camera {
...
     virtual Status getImagerType(ImagerType &imagerType) const = 0;
+
+    /**
+     * @brief Save ini file to json format
+     * @param pathFiles - Path to ini files
+     * @param savePathFile - Path to save json file
+     * @return Status
+     */
+    virtual aditof::Status
+    saveDepthParamsToJsonFile(const std::vector<std::string> &pathFiles,
```
Comment (dNechita) [inline sdk/include/aditof/camera.h:545]: What is the reason for adding parameter pathFiles?

File: sdk/src/cameras/itof-camera/camera_itof.cpp [hunk 2]
```diff
@@ -49,10 +49,12 @@
 #else
 #include <aditof/log.h>
 #endif
+#include <cctype>
```
Comment (rbudai98) [inline sdk/src/cameras/itof-camera/camera_itof.cpp:52]: This might be unnecessary

Comment (VHolonec) [reply sdk/src/cameras/itof-camera/camera_itof.cpp]: Solved

File: sdk/src/cameras/itof-camera/camera_itof.cpp [hunk 3]
```diff
@@ -1549,6 +1552,70 @@ aditof::Status CameraItof::parseJsonFileContent() {
...
+    cJSON_AddItemToObject(rootjson, "depthParams", depthParamjson);
+
+    for (auto pfile = m_ini_depth_map.begin(); pfile != m_ini_depth_map.end();
+         pfile++) {
+        size_t lastUnderscorePos = pfile->second.find_last_of("_");
+        if (lastUnderscorePos == std::string::npos) {
+            continue;
+        }
+        size_t dotPos = pfile->second.find_last_of(".");
+        std::string modeName = pfile->second.substr(
```
Comment (dNechita) [inline sdk/src/cameras/itof-camera/camera_itof.cpp:1571]: I believe that m_ini_depth_map already contains the name of the mode (it's the key and can be obtained with pfile->first). So there is no need to extract it again from the file name.

Comment (VHolonec) [reply sdk/src/cameras/itof-camera/camera_itof.cpp]: Yes indeed, pfile->first contain the mode name.

**Review by dNechita — APPROVED** [review]

**Pull request merged**